raiden/network/proxies/token_network.py

```diff
@@ -2393,25 +2390,11 @@ def _settle(
                     locksroot=partner_locksroot,
                 )
 
-                participants = get_channel_participants_from_open_event(
-                    token_network=self,
-                    channel_identifier=channel_identifier,
-                    contract_manager=self.contract_manager,
-                    from_block=self.metadata.filters_start_at,
-                )
-                if not participants:
-                    msg = (
-                        f"The provided channel identifier {channel_identifier} "
-                        f"does not exist on-chain."
-                    )
-                    raise RaidenUnrecoverableError(msg)
-
-                if self.node_address not in participants:
-                    msg = (
-                        f"Settling a channel in which the current node is not a participant "
-                        f"of is not allowed."
-                    )
-                    raise RaidenUnrecoverableError(msg)
+                # The channel_identifier and the address of the participants
+                # can not be verified because querying for the open event would
+                # take prohibitly long (see #6106). Instead it is assumed that
```

Comment:
```suggestion
                # take prohibitively long (see #6106). Instead it is assumed that
```

Reply:
I always amaze myself on how creative I can be with different spellings of a word. Thanks